source/database-connections/save-mongodb-connection.txt

```diff
@@ -0,0 +1,106 @@
+
+.. _rm-save-mongodb-connection:
+
+============================================
+Save a MongoDB Connection
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To start a migration job, Relational Migrator must connect to your MongoDB
+database using an account with :authrole:`readWrite` access. You can save a new
+MongoDB connection from the Relational Migrator :guilabel:`Connections` page,
+or when creating or modifying a data migration job.
+
+Before you Begin
+----------------
+
+- To save a connection to a MongoDB Atlas cluster, you must be :ref:`logged in
+  <rm-atlas-log-in>` with Atlas.
```

Comment:
This feature has unfortunately stopped working due to changes in the Atlas API, and we have "temporarily" hidden the ability to select Atlas clusters from a dropdown. (I say "temporarily" but we don't know if/when we'll be able to re-enable it). So we can remove this section for the time being.